runtime/moonriver/src/lib.rs

```diff
@@ -1330,7 +1351,27 @@ impl pallet_multisig::Config for Runtime {
 	type WeightInfo = moonbeam_weights::pallet_multisig::WeightInfo<Runtime>;
 }
 
-construct_runtime! {
+/// Custom macro to enable some pallets only when compiled with `force-debug` feature
+#[macro_export]
+macro_rules! construct_moonriver_runtime {
+	{$($common_pallets:tt)*} => {
+		#[cfg(feature = "force-debug")]
+		construct_runtime! {
+			$($common_pallets)*
+
+			// Aditional pallets with `force-debug` feature:
+			// Sudo was previously index 40
```

Comment:
For safety, it would be better to use a different index. Because the sudo key storage was not removed on moonriver live storage

Reply:
Changed index to 240